Falco AWS Collaboration blog (#856)
falcosecurity/falco-website: content/en/blog/Falco Community and AWS Collaboration_ Driving Innovation in Cloud-Native Security (1).md

---
title: "Falco Community and AWS Collaboration: Driving Innovation in Cloud-Native Security"
linktitle: "How Open-Source Innovation is Shaping the Future of Cloud-Native Security"
date: 2023-03-15
author: Jacqueline Salinas
slug: falco-aws-collaboration
---

> Today, the Falco community announced general availability of an integration to help users easily route cloud-native security events from Falco to [Amazon Security Lake](https://aws.amazon.com/security-lake/) using Falcosidekick. Falco, an open source project, is the default threat detection engine for cloud and containers, with more than 60 million downloads. Developed by Sysdig and contributed to the CNCF, Falco is an Incubation-level project. Falcosidekick is a companion tool that forwards and exports Falco events to and from other services using the standard Open Cybersecurity Schema Framework (OCSF) format. Amazon Security Lake can consolidate security information from various sources including on-premises, cloud, and custom sources, and save it in a specialized data lake in your AWS account automatically. This solution can help you gain a comprehensive view of security data throughout your organization, ultimately enhancing the protection of your workloads, applications, and data.

Author: ```suggestion
Today, the Falco community announces general availability of its Falcosidekick integration with Amazon Security Lake. With this integration users can easily route cloud-native security events from Falco to[ Amazon Security Lake](https://aws.amazon.com/security-lake/). Falco, an open source project, is the default threat detection engine for cloud and containers, with more than 60 million downloads. Developed by Sysdig and contributed to the CNCF, Falco is an Incubation-level project. Falcosidekick is a companion tool that forwards and exports Falco events to and from other services using the standard Open Cybersecurity Schema Framework (OCSF) format. Amazon Security Lake can consolidate security information from various sources including on-premises, cloud, and custom sources, and save it in a specialized data lake in your AWS account automatically. This solution can help you gain a comprehensive view of security data throughout your organization, ultimately enhancing the protection of your workloads, applications, and data.
```

> With the new Falcosidekick integration, receiving and uploading events into Amazon Security Lake is easier for developers and brings together Falco events with other security data sources, making it easier to use Amazon Security Lake. Falco was a key partner when Amazon Security Lake was previewed last year and as the tool prepares to be generally available, Falco facilitates capturing data from additional sources and enables consistent threat detection across workloads and infrastructure. 

Author: ```suggestion
With the new Falcosidekick integration, receiving and uploading events into Amazon Security Lake is easier for developers and brings together Falco events with other security data sources, making it easier to use Amazon Security Lake. Falco was a key partner when [Amazon Security Lake](https://aws.amazon.com/security-lake/) was previewed last year and as the tool prepares to be generally available, Falco facilitates capturing data from additional sources and enables consistent threat detection across workloads and infrastructure.
```

> In early 2022, using Falcosidekick, a companion tool that forwards and exports Falco events to/from other services, the Falco community created a new extension using the new plug-in framework that enables users to extend Falco's capabilities to capture data from sources beyond Linux system calls and Kubernetes audit logs. As organizations manage critical data across multiple clouds, consistent threat detection is vital across their distributed environments. Additional plug-ins will enable businesses to use a uniform threat detection language and close security gaps by utilizing standardized policies for workloads and infrastructure. Today, a broad range of plug-ins are available for use, with contributors being able to develop additional ones.

Author: ```suggestion
In early 2022, using Falcosidekick, a companion tool that forwards and exports Falco events to/from other services, the Falco community created a new extension using the new plug-in framework that enables users to extend Falco's capabilities to capture data from sources beyond Linux system calls and Kubernetes audit logs. As organizations manage critical data across multiple clouds, consistent threat detection is vital across their distributed environments. Additional plug-ins will enable businesses to use a uniform threat detection language and close security gaps by utilizing standardized policies for workloads and infrastructure. Today, a broad range of plug-ins are available for use, with contributors being able to develop additional ones.

```
